.github/workflows/auth-testing.yml

```
@@ -0,0 +1,232 @@
+        continue-on-error: true
+
+      - name: Check for hardcoded secrets
+        run: |
+          echo "Checking for potential hardcoded secrets..."
+          # Look for potential API keys, tokens, etc.
+          if grep -r -E "(api[_-]?key|secret|token|password)" --include="*.ts" --include="*.tsx" --include="*.js" --include="*.jsx" apps/ packages/ | grep -v "test" | grep -v "__tests__" | grep -v ".spec." | grep -v "mock"; then
+            echo "⚠️ Potential hardcoded secrets found. Please review:"
+            grep -r -E "(api[_-]?key|secret|token|password)" --include="*.ts" --include="*.tsx" --include="*.js" --include="*.jsx" apps/ packages/ | grep -v "test" | grep -v "__tests__" | grep -v ".spec." | grep -v "mock" || true
+          else
+            echo "✅ No obvious hardcoded secrets found"
+          fi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)